test/integration/crcsuite/prepare.go

```diff
@@ -56,6 +58,60 @@ func DownloadBundle(bundleLocation string, bundleDestination string) (string, er
 	return filename, nil
 }
 
+func CopyFilesToTestDir() {
+
+	pwd, err := os.Getwd()
+	if err != nil {
+		fmt.Printf("Error retrieving current dir: %s", err)
+		os.Exit(1)
+	}
+	pDir := path.Dir(pwd)
+	pDir = path.Dir(pDir)
+	pDir = path.Dir(pDir)
+	dataDir := filepath.Join(pDir, "testdata")
+	fmt.Printf("dataDir: %s\n", dataDir)
+
+	files, err := ioutil.ReadDir(dataDir)
+	if err != nil {
+		fmt.Printf("Error occured loading data files: %s", err)
+		os.Exit(1)
+	}
+
+	for _, file := range files {
+
+		sFileName := filepath.Join(dataDir, file.Name())
+		destLoc, _ := os.Getwd()
```

Comment:
This is not going to change while looping over the files, is it? This could be moved out of the loop.

Reply:
Good point, `destLoc` is independent of the `file`. Thanks